colink/__init__.py

```diff
@@ -1,13 +1,13 @@
 from typing import Tuple
 from colink.colink_pb2 import *
-from colink.colink_pb2_grpc import *
+from colink.colink_pb2_grpc import CoLinkStub, CoLinkServicer
```

Comment:
In colink_pb2_grpc there is also a CoLink class, to resolve conflict we can't import *.